images/singleuser/Dockerfile

```diff
@@ -142,7 +142,14 @@ RUN sed -i -e '/^R_LIBS_USER=/s/^/#/' /etc/R/Renviron && \
 ENV VENV_DIR /srv/paws
 RUN install -d -o ${NB_USER} -g ${NB_USER} ${VENV_DIR}
 
-ENV PATH=/srv/paws/pwb:/srv/paws/bin:/srv/paws:$PATH
+ENV PATH=/srv/paws/pwb:/srv/paws/bin:/srv/paws:/srv/julia/bin:$PATH
```

Comment:
If you're setting $JULIA_PATH, you should probably reference that here instead of hardcoding it again